Makefile.am

```diff
@@ -93,6 +94,21 @@ mcs-do-compiler-tests:
 bootstrap-world: compiler-tests
 	$(MAKE) install
 
+install:
+	echo $(SUBDIRS)
+	for mydir in $(SUBDIRS) ; do \
+		do_install=1 ; \
+		for noinstdir in $(noinst_SUBDIRS) ; do \
+			if [ "x$${noinstdir}" = "x$${mydir}" ] ; then \
+				do_install=0 ; \
+				break ; \
+			fi \
+		done ; \
+		if [ "$${do_install}" = "1" ] ; then \
+			(cd $${mydir} && ${MAKE} install) \
```

Comment:
should this be `$(MAKE) install` instead or does it not matter?

Reply:
It appears not to matter.